package-lock.json

```diff
@@ -5524,6 +5524,12 @@
       "integrity": "sha512-dRLjCWHYg4oaA77cxO64oO+7JwCwnIzkZPdrrC71jQmQtlhM556pwKo5bUzqvZndkVbeFLIIi+9TC40JNF5hNQ==",
       "dev": true
     },
+    "node_modules/@types/lodash": {
```

Comment:
Why do you have package-lock changes without package changes?

Reply:
- Removed: 6f4005a621b4259770a6dbaa14b1716c258d87d1